crypto/fipsmodule/rsa/rsa_impl.c

```diff
@@ -72,13 +72,7 @@
 #include "../delocate.h"
 #include "../rand/fork_detect.h"
 
-
-int rsa_check_public_key(const RSA *rsa) {
-  if (rsa->n == NULL || rsa->e == NULL) {
-    OPENSSL_PUT_ERROR(RSA, RSA_R_VALUE_MISSING);
-    return 0;
-  }
-
+int rsa_check_key_common(const RSA *rsa, const rsa_key_type_t key_type) {
```

Comment:
Should this also check for the modulus being too small?

Reply:
perhaps. i tried to hew as closely as possible to existing logic and code paths (while still allowing us to use stripped keys, which motivated this PR), so i did not add additional validation.

if AWS-LC folks would like such validation in this code path and can point me towards some guidance on modulus size (i'd guess that this is mostly an issue w/r/t key generation?), i'd be happy to take a stab at adding it, perhaps in another PR to keep the scope of this one tight.

Reply:
what would be the threshold for too small modulus? I'm afraid that if we set the threshold to something reasonable like 1024 bits we'll break something because I bet there are 512/768 bit moduli still in use.

If you are referring to the check "is n > e" this check is done further below in the function.

Reply:
reached consensus offline on this, no new tests added.